### PR TITLE
retain changes after saving

### PR DESCRIPTION
**Pull request by goemen**

### CODE REVIEW

File: frontend/src/features/projects/common/projectConverter.ts [hunk 1]
```diff
@@ -208,15 +211,19 @@ export const toApiProperty = (property: IProperty): IApiProperty => {
         parcelId: property.propertyTypeId === 0 ? property.id : undefined,
         buildingId: property.propertyTypeId === 1 ? property.id : undefined,
         value: property.netBook,
-        fiscalYear: property.netBookFiscalYear ?? getCurrentFiscalYear(),
+        fiscalYear: !useCurrentFiscal
+          ? property.netBookFiscalYear ?? getCurrentFiscalYear()
```
Comment (Fosol) [inline frontend/src/features/projects/common/projectConverter.ts:215]: What does `getCurrentFiscalYear()` return?

Comment (Fosol) [reply frontend/src/features/projects/common/projectConverter.ts:215]: Do you need something similar for `Evaluations`?

**Review by Fosol — APPROVED** [review]

File: frontend/src/features/projects/common/projectConverter.ts (hunk 1 above)
Comment (goemen) [reply frontend/src/features/projects/common/projectConverter.ts:215]: The evaluation are working just fine

Comment (goemen) [reply frontend/src/features/projects/common/projectConverter.ts:215]: > What does `getCurrentFiscalYear()` return?

@Fosol returns the current fiscal year

**Pull request merged**